src/app/saga.ts

```diff
@@ -204,6 +204,7 @@ export interface RemoteConfigValues {
   networkTimeoutSeconds: number
   dappFavoritesEnabled: boolean
   celoNews: CeloNewsConfig
+  twelveWordSeedPhraseEnabled: boolean
```

Comment:
nit: sorry I realize we're using `mnemonic` throughout the code base, I'd suggest we use the same terminology everywhere in this PR too.
```suggestion
  twelveWordMnemonicEnabled: boolean
```

Reply:
Changed in 3536da4d134b87d614431868fec162e397310245!